src/ONYXKEYS.js

```diff
@@ -71,6 +71,9 @@ export default {
     // Contains all the users settings for the Settings page and sub pages
     USER: 'user',
 
+    // Contains user login list
```

Comment:
Can we be more descriptive here? Something like:

```suggestion
    // Contains metadata (partner, login, validation date) for all of the user's logins
```